ImperatorToCK3/Source/CK3/Character/CK3Character.cpp

```diff
@@ -45,6 +47,11 @@ void CK3::Character::initializeFromImperator(
 		if (traitMatch) traits.insert(*traitMatch);
 	}
 
+	if (!imperatorCharacter->getNickname().empty())
+	{
+		auto nicknameMatch = nicknameMapper.getCK3NicknameForImperatorNickname(imperatorCharacter->getNickname());
+		if (nicknameMatch) nickname = *nicknameMatch;
```

Comment:
To confirm, all nicknames that have no translation will not carry over - so people with those nicks won't have one in CK3?

Reply:
Yeah, that's the plan.